spec/client_server_spec.rb

```diff
@@ -62,7 +76,7 @@ def recording_server(replies = {})
 
     it 'should emit :disconnected when the connection disconnects' do
       recording_server { |server|
-        client = EM::Hiredis::Client.new('redis://localhost:6381/9')
+        client = EM::Hiredis::Client.new('redis://127.0.0.1:6381/9')
```

Comment:
I don't think these should be necessary and it makes the review harder to follow. Any chance we could remove them?
Happy to pair to try and get them working on your machine if that's the problem

Reply:
It doesn't work for me locally with `localhost`. As far as I know it's a known bug with MacOS. I am happy to pair if we can resolve this!